project/Dependencies.scala

```diff
@@ -4,7 +4,8 @@ import sbtassembly.AssemblyPlugin.autoImport.{MergeStrategy, assemblyMergeStrate
 import sbtassembly.PathList
 
 object Dependencies {
-  val awsVersion = "1.11.879"
+  val awsVersion1 = "1.11.892"
+  val awsVersion2 = "2.15.19"
```

Comment:
minor but could be more like 
```suggestion
  val awsV1Version = "1.11.892"
  val awsV2Version = "2.15.19"
```

Reply:
That was my first thought but then it's like version1version.  A bit like a PIN number being a number number.
But if you think it's clearer I'll change it.

Reply:
I just felt like it doesn't make sense the existing way - it makes sense to me to have a version number for each of version 1 and version 2.
But either way it's a minor point.